### PR TITLE
Answers to Search (5): Final touches for renaming Answers to Search

### PR DESCRIPTION
**Pull request by alextaing**
Updated docs to reflect change from Answers to Search
Decided to keep search-core agent info to be called `ANSWERS-CORE`
Address other nits

### CODE REVIEW

File: src/models/searchservice/response/Source.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@
  * @public
  */
 export enum Source {
-  /** The result is from an Answers Knowledge Graph. */
+  /** The result is from a Search Knowledge Graph. */
```
Comment (oshi97) [inline src/models/searchservice/response/Source.ts:7]: I'd probably have this just `The result is from Knowledge Graph`, "Answers Knowledge Graph" was a little confusing

Comment (alextaing) [reply src/models/searchservice/response/Source.ts]: like it 👍

File: src/services/SearchService.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { VerticalSearchRequest } from '../models/searchservice/request/VerticalS
 import { VerticalSearchResponse } from '../models/searchservice/response/VerticalSearchResponse';
 
 /**
- * A service which performs Yext Answers searches
+ * A service which performs Yext Search
```
[the remark this answers is not in the document]
Comment (alextaing) [reply src/services/SearchService.ts]: fixed!

File: src/models/autocompleteservice/AutocompleteResponse.ts [hunk 3]
```diff
@@ -41,7 +41,7 @@ export interface FilterSearchResponse {
     /** An array of {@link AutocompleteResult}s. */
     results: AutocompleteResult[]
   }[],
-  /** ID of the account associated with this Answers experience */
+  /** ID of the account associated with this Search experience */
```
[the remark this answers is not in the document]
Comment (alextaing) [reply src/models/autocompleteservice/AutocompleteResponse.ts]: fixed!

File: src/validation/ApiResponseValidator.ts [hunk 4]
```diff
@@ -2,7 +2,7 @@ import { ApiResponse } from '../models/answersapi/ApiResponse';
 import { SearchError } from '../models/answersapi/SearchError';
 
 /**
- * Determines whether or not an API response can be used to construct an search-core response
+ * Determines whether or not an API response can be used to construct a search-core response
```
[the remark this answers is not in the document]
Comment (alextaing) [reply src/validation/ApiResponseValidator.ts]: fixed!

File: src/models/core/AdditionalHttpHeaders.ts [hunk 5]
```diff
@@ -20,7 +20,7 @@ export interface ClientSDKHeaderValues {
   [agent: string]: string | undefined,
   /**
    * The ANSWERS_CORE agent should not be supplied. Instead, it will be automatically added to the
-   * header and populated with the version of Answers Core being used.
+   * header and populated with the version of Search Core being used.
    */
   ANSWERS_CORE?: never
```
[the remark this answers is not in the document]
Comment (alextaing) [reply src/models/core/AdditionalHttpHeaders.ts]: updated!

**Review by yen-tt — COMMENTED** [review]
couple more things to update!
src/models/index.ts have `// Answers API models`
probably rename src/models/answersapi folder

File: src/services/SearchService.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (oshi97) [reply src/services/SearchService.ts]: I tried for ~5ish minutes and couldn't find anything though.

**Review by oshi97 — APPROVED** [review]

**Pull request merged**